push.go

```
@@ -35,15 +34,33 @@ func (cmd *PushCommand) Run() error {
 		return err
 	}
 
-	formats, err := client.FormatsList(1, 25)
-	if err == nil {
-		err = sources.setFormats(formats)
-		if err != nil {
+	formatMap, _ := GetFormats(client)
```

what is swallowed here? if it is an error, always handle it!